app/models/concerns/multipage_pdf_renderer.rb

```diff
@@ -0,0 +1,35 @@
+require 'open3'
+
+# https://github.com/mileszs/wicked_pdf/issues/339
+class MultipagePdfRenderer
+  def self.combine(documents, options)
+    outfile = ::WickedPdf::WickedPdfTempfile.new('multipage_pdf_renderer.pdf')
+
+    tempfiles = documents.each_with_index.map do |doc, index|
+      file = ::WickedPdf::WickedPdfTempfile.new("multipage_pdf_doc_#{index}.html")
+      file.binmode
+      file.write(doc)
+      file.rewind
+      file
+    end
+
+    filepaths = tempfiles.map{ |tf| tf.path.to_s }
+
+    wickedPdf = WickedPdf.new
+    binary = wickedPdf.send(:find_wkhtmltopdf_binary_path)
+    options = wickedPdf.send(:parse_options, options)
```

Comment:
Is this Ruby's `send` to send the `find_wkhtmltopdf_binary_path` message? Is there any reason to not directly invoke it (ie, `wickedPdf.find_wkhtmltopdf_binary_path`)?

Reply:
It's necessary to invoke private messages.